tests/behaviour/events_test.py

```diff
@@ -0,0 +1,43 @@
+import asyncio
+
+import pytest
+from starkware.starknet.compiler.compile import compile_starknet_files
+from starkware.starknet.testing.state import StarknetState
+
+from tests.utils import CAIRO_PATH
+from warp.yul.main import transpile_from_solidity
+from warp.yul.starknet_utils import deploy_contract, invoke_method
+
+test_dir = __file__
+
+
+@pytest.mark.asyncio
+async def test_events():
+    contract_file = test_dir[:-8] + ".cairo"
```

Comment:
Behaviour tests don't actually need cairo-files. We can get this cairo files by transpiling from solidity. Furthermore, it is useful as it allows to keep .sol and .cairo files out of sync. You can take a look at for-loop-no-condition_test.py to see how it can be done